pyvcloud/vcd/client.py

```diff
@@ -610,6 +626,9 @@ def logout(self):
         uri = self._uri + '/session'
         return self._do_request('DELETE', uri)
 
+    def is_loogedin_user_sys_admin(self):
```

Comment:
As this method going to be used extensively at several places, please shorten the name. is_sysadmin or better?

Reply:
is_user_sysadmin? I don't think, just because the method will be used in many places, it should be shortened. 
 If we are shortening it, we should add docstring to describe the method.

Reply:
I agree that the name of the variable is a mouthful.
What we really want to ascertain is that the current context of the logged in user is that of sys_admin.

client.is_in_sysadmin_context() makes more sense?

Reply:
Anusuya, it is better to have short method names in general. Given that this method is going to be used extensively, imho, it is okay to give it a second thought.

Reply:
@sahithi okay.